src/main/java/Byte.java

```diff
@@ -0,0 +1,116 @@
+import java.util.Scanner;
+
+public class Byte {
+    private static final int MAX_TASKS = 100;
+    private static final Task[] tasks = new Task[MAX_TASKS];
+    private static int taskCount = 0;
+
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        runByte(scanner);
+        scanner.close();
+    }
+
+    public static void runByte(Scanner scanner){
+        printWelcomeMessage();
+        while(true){
+            String userInput = scanner.nextLine();
+            printLineSeparator();
+            if (processUserInput(userInput)) {
+                break;
+            }
+        }
+    }
+
+    public static boolean processUserInput(String userInput){
+        if (userInput.equals("bye")) {
+            printGoodbyeMessage();
+            return true;
+        } else if (userInput.startsWith("mark ")) {
```

Comment:
else if should be on a new line after brackets

Reply:
I followed according to "https://se-education.org/guides/conventions/java/basic.html" though.